lib/digitalocean/bootstrap/oks-firewall.tf

```diff
@@ -0,0 +1,22 @@
+# due to tags, (applied on kube cluster and pools), all droplets (scaled) will me automatically attached to the firewall below
+resource "digitalocean_tag" "cluster_tag" {
+  name = var.oks_master_name
```

Comment:
can you please same tags we've set for AWS as well

Reply:
I also advise you to look on how it has been done on AWS

Reply:
tags doesn't works with the same behavior of aws
They are very linked with do firewalls